core/src/main/scala/com/yahoo/maha/core/query/druid/DruidQueryGenerator.scala

```diff
@@ -1037,6 +1076,18 @@ class DruidQueryGenerator(queryOptimizer: DruidQueryOptimizer
             columnAlias)
         case DruidFilteredListRollup(_, _, _) =>
           getFilteredListAggregatorFactory(dataType, rollup, alias)
+        case DruidHyperUniqueTimeBucketedRollup(fieldName) if (forOuterQuery) =>
+          getAggregatorFactory(
+            dataType,
+            SumRollup,
+            alias,
+            columnAlias)
+        case DruidHyperUniqueTimeBucketedRollup(fieldName) =>
+          val hyperUniqueOutputName = s"_hyperUniqueOutput_$fieldName"
+          postAggregatorList += new HyperUniqueFinalizingPostAggregator(columnAlias, hyperUniqueOutputName)
+          getHyperUniqueAggregatorFactory(dataType, hyperUniqueOutputName, fieldName)
+        case DruidHyperUniqueRollup(fieldName) if (forOuterQuery) =>
```

Comment:
Do you have an updated test to test this change?

Reply:
I included `{"field": "Unique Ad IDs"}`, which uses a `DruidHyperUniqueRollup`, in the test case below to cover this code path and ensure outer query aggregations for this rollup are correctly generated.

Reply:
I included `{"field": "Unique Ad IDs"}`, which uses a `DruidHyperUniqueRollup`, in the test case below to cover this code path and ensure outer query aggregations for this rollup are correctly generated.